AndyDorokhin/HW8/task3.py

```diff
@@ -8,3 +8,26 @@
     three functions for finding areas, into the main program. The basic logic of the
     program is executed in the main module).
 """
+from task3_helper import rectangle_area, triangle_area, circle_area
```

Comment:
good job
look at this version of code

def rectangle_area():
    a = get_float_input("Enter a: ")
    b = get_float_input("Enter b: ")
    return a * b

def triangle_area():
    base = get_float_input("Enter base: ")
    height = get_float_input("Enter height: ")
    return 0.5 * base * height

def circle_area():
    radius = get_float_input("Enter radius: ")
    return 3.14 * radius**2

def get_float_input(prompt):
    while True:
        try:
            value = float(input(prompt))
            return value
        except ValueError:
            print("Invalid input. Please enter a valid number.")

def main():
    print("1. Rectangle")
    print("2. Triangle")
    print("3. Circle")
    choice = input("Enter your choice: ").strip()
    if choice == '1':
        area = rectangle_area()
        print(f"Area of rectangle: {area:.2f}")
    elif choice == '2':
        area = triangle_area()
        print(f"Area of triangle: {area:.2f}")
    elif choice == '3':
        area = circle_area()
        print(f"Area of circle: {area:.2f}")
    else:
        print("Invalid choice")

if __name__ == "__main__":
    main()